app/models/time_record.rb

```diff
@@ -41,10 +41,16 @@ def belongs_to_user?(user_id)
   end
 
   def self.total_time
-    total_time = all.sum(:spent_time)
-    if active_task = all.find_by("time_start IS NOT NULL")
+    total_time = all.map(&:spent_time).inject(0, :+)
```

Comment:
Can you describe what is the difference between old and new code? For me the just the same but new is a bit complicated.

Reply:
the old one doesn't work fine because I have the query and if I use `all.sum` that adds this `sum` to the query. And if this query returns several time records with the same `spent_time` this records would be dropped by the condition 'distinct'